src/components/Explore/Explore.tsx

```diff
@@ -116,17 +117,23 @@ const Explore: React.FunctionComponent<{
 
   const currentMetricName = getMetricName(currentMetric);
 
-  const currentLocation = useMemo(() => findLocationForFips(fips), [fips]);
-  const autocompleteLocations = useMemo(() => getAutocompleteLocations(fips), [
-    fips,
+  const currentLocations = useMemo(() => fipsList.map(findLocationForFips), [
+    fipsList,
   ]);
+  const autocompleteLocations = useMemo(
+    () => getAutocompleteLocations(fipsList[0]),
```

Comment:
in what situations does the fipsList change after the fact?

Reply:
The locations in the dropdown are sorted depending on the current location (this is the order if you don't type anything):
 - if you are in a state page, it shows you states first, then counties in the current state, then all the other counties
- if you are on a county page, it shows you other counties in the state, then states, and then other counties.

As soon as you start typing the matching selections will show up. Since we don't need the list to be re-ordered when other variables change (normalize, adding locations, etc), we use the `useMemo` hook to avoid the list from being recomputed.